src/audio/module_adapter/module/modules.c

```diff
@@ -143,6 +143,7 @@ static int modules_init(struct processing_module *mod)
 					(struct module_interface *)md->module_adapter;
 
 		ret = mod_in->init(mod);
+		mod->priv.ops = mod_in;
```

Comment:
Will need a commit message too.

Reply:
@RanderWang even if this is missing, everywhere we call mod_in in prepare/process, we always assign before using. What error does this fix?

Reply:
Do we assign it before using in prepare or process ? For built-in module, it is defined by the module declaration, but for loadable module it is defined by a default interface, so we need to overwrite it by private loadable module interface
````c
/* Processing Module Adapter API*/
static const struct module_interface interface = {
        .init = modules_init,
        .prepare = modules_prepare,
        .process_raw_data = modules_process,
        .set_processing_mode = modules_set_processing_mode,
        .get_processing_mode = modules_get_processing_mode,
        .set_configuration = modules_set_configuration,
        .get_configuration = modules_get_configuration,
        .reset = modules_reset,
        .free = modules_free,
};
````

Reply:
I find this change is not correct. The overwrite will lost some functions

Reply:
changed